airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```
@@ -1698,7 +1698,9 @@ definitions:
       page_size:
         title: Page Size
         description: The number of records to include in each pages.
-        type: integer
+        anyOf:
+          - type: integer
+          - type: string
         examples:
           - 100
           - "100"
```

```suggestion
          - "{{ config['page_size'] }}"
```

Funny that this was here before, but I like the example in `OffsetIncrement` better. And the example seemingly wouldn't actually work before your change? Haven't tested myself yet.

---

In my local testing, using a `"100"` didn't work - only literal ints did.